nodejs/week3/homework/warmup/index.js

```diff
@@ -0,0 +1,67 @@
+const express = require("express");
+const app = express();
+
+app.use(express.urlencoded({extended: true
+}));
+app.use(express.json());
+
+app.get("/", (req, res) => res.send("nodejs week3 homework"));
+
+app.get("/calculator/:method", (req, res) => {
+    if (req.params.method === "add") {
+      let sumResult = addition(req);
+      res.send(sumResult);
```

Comment:
```suggestion
      res.send(`${sumResult}`);
```
and similar for the rest of responses